src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,195 @@
+    public static final String INFO_READ
+            = "read [path] [file-name] - Считывает файл по указанному пути и "
+            + "выводит текст в консоль;";
+    public static final String INFO_INFO
+            = "info [path] [file-name] - Выводит краткую информацию по указанному файлу;";
+    public static final String INFO_HELP
+            = "help - Выводит в консоль все доступные команды и информацию к ним;";
+    public static final String INFO_HELP_COMMAND
+            = "help [command] - Выводит в консоль информацию по указанной команде;";
+    public static final String INFO_EXIT
+            = "exit - Завершение работы программы.";
+
+    public static void chooseCommand(String input) {
+        String[] command = input.split(" ");
+        switch (command[0]) {
+            case "create":
+                create(new File(command[1], command[2]));
+                break;
+            case "read":
+                System.out.println("Содержимое файла " + "\'" + command[2] + "\' :");
+                System.out.print(read(new File(command[1], command[2])));
+                break;
+            case "info":
+                info(new File(command[1], command[2]));
+                break;
+            case "help":
+                help(command);
+                break;
+            case "exit":
+                System.exit(0);
+                break;
+            default:
+                noCommand(input);
+                break;
+        }
+    }
+
+    public static void create(File file) {
+        try {
+            boolean created = file.createNewFile();
+            if (created) {
+                System.out.println("Файл создан");
+            } else {
+                System.out.println("Файл уже существует. Хотите перезаписать? Y - Да, N - Нет");
+                switch (new Scanner(System.in).nextLine().toUpperCase()) {
+                    case "Y":
+                        file.delete();
+                        System.out.println(file.createNewFile() ? "Файл перезаписан" :
+                                "Системе не удалось сделать перезапись");
+                        break;
+                    case "N":
+                        System.out.println("Вы отменили перезапись файла");
+                        break;
+                    default:
+                        System.out.println("Неизвестная команда!");
+                        break;
+                }
+            }
+        } catch (IOException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public static String read(File file) {
+        String read = "";
+        try (FileReader fileReader = new FileReader(file);
+                Scanner readScanner = new Scanner(fileReader)) {
+            while (readScanner.hasNextLine()) {
+                read += readScanner.nextLine() + "\n";
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("Такого файла по заданному пути не существует");
```

Comment:
do not swallow the exception. Remember about the picture on the lesson. Service layer - for business logic (should throw an exception if there is some edge situation). Main class - is only an entry point. Catch the exception on some additional level between services and the Main class. Or right in the Main class